dll/win32/comctl32/listbox.c

```diff
@@ -1945,7 +1951,11 @@ static LRESULT LISTBOX_HandleHScroll( LB_DESCR *descr, WORD scrollReq, WORD pos
         case SB_THUMBTRACK:
```

Comment:
This can be received when scroll is either down horizontally or vertically, so your change below is incomplete.